DigitalLearningSolutions.Web/Views/Register/Summary.cshtml

```diff
@@ -83,6 +85,21 @@
           </a>
         </dd>
       </div>
+      @foreach (EditCustomFieldViewModel customField in ViewBag.CustomFields) {
+        <div class="nhsuk-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            @(customField.CustomPrompt + (customField.Mandatory ? "" : " (optional)"))
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @customField.Answer
+          </dd>
+          <dd class="nhsuk-summary-list__actions">
+            <a class="nhsuk-link" asp-action="LearnerInformation">

```

Comment:
There seem to be loads of these nhs classes that are unknown. It doesn't recognize nhsuk-table__caption or nhsuk-table__head for me despite them being in the html copied from the nhs component definition

Reply:
Yeah, I expect the NHS is happy with the default styling for some components but still recommends adding a class to them in case that changes in the future.

Reply:
Ah very interesting - I can't see the link style used in any of the official examples: https://service-manual.nhs.uk/design-system/components/summary-list

Not sure about the Table styles - maybe we've got the old version of the NHS styles library?  👀

Reply:
Aha, I must have got it from [here](https://github.com/tomdoughty/nhsuk.base-application/blob/main/src/nhsuk.base-application/Views/ExampleForm/Summary.cshtml) then when I was first making the summary page. You're right that it doesn't appear to be anywhere in the design system docs, it can probably be deleted then.